### PR TITLE
feat: add emoji input for subject content

### PR DESCRIPTION
**Pull request by liuliangsir**
- add support for commit message with emoji
- add access support for emoji with local access and remote access
- add some npm scripts
- adjust project directory structure
- fix a certain typo
- fix code style

### CODE REVIEW

File: config/emojis.json [hunk 1]
```diff
@@ -0,0 +1,338 @@
+[
+  {
+    "emoji": "🎨",
```
Comment (leonardoanalista) [inline config/emojis.json]: I like the idea of emojis. Does this list need to live in this repo? 
Is there a way to easily extend the emoji list?

**Comment by leonardoanalista** [comment]
lgtm, we merge when tests pass. 👍

File: config/emojis.json (hunk 1 above)
Comment (liuliangsir) [reply config/emojis.json]: Yes, It could be replaced with [an external link](https://github.com/carloscuesta/gitmoji-cli/blob/master/src/utils.js#L4)

Comment (liuliangsir) [reply config/emojis.json]: hey, it appears to be a breaking change.

**Pull request closed without merging**